cpu/nrf52/Kconfig

```diff
@@ -15,6 +15,7 @@ config CPU_FAM_NRF52
     select HAS_CORTEXM_MPU
     select HAS_CPU_NRF52
     select HAS_PERIPH_I2C_RECONFIGURE
+    select HAVE_NRF5X_RADIO
```

Comment:
What about `HAS_RADIO_NRF802154` `select`ing this instead?

Reply:
Makes sense!